### PR TITLE
docs: formatting, and note about hardened-chromium settings in post-install

### PR DESCRIPTION
**Pull request by mintpilo**

### CODE REVIEW

File: POSTINSTALL-README.md [hunk 1]
```diff
@@ -127,6 +127,11 @@ To validate your secureblue setup, run:
 ```
 ujust audit-secureblue
 ```
+
+## Optional: `hardened-chromium` Flags
+The included hardened-chromium browser has some additional settings in `chrome://flags` you *may* want to set for additional hardening, and convenience. (At the cost of *some* functionality.)
```
Comment (RoyalOughtness) [inline POSTINSTALL-README.md]: > (At the cost of *some* functionality.)

Maybe "That can cause functionality issues in some cases"

File: README.md [hunk 2]
```diff
@@ -78,11 +78,13 @@ Sponsorship options are on the [Donate](DONATE.md) page. All donations are appre
...
 ## Rebasing (Recommended)
-*Note: if you don't already have a Fedora Atomic installation, use a Fedora Atomic iso that matches your secureblue target image to install one. If you want to use a secureblue Silverblue image, start with the Fedora Silverblue iso, Kinoite for Kinoite, and Sericea (Sway Atomic) for Sericea and all the Wayblue images.*
+> [!NOTE] 
+> If you don't already have a Fedora Atomic installation, use a Fedora Atomic ISO that matches your secureblue target image to install one. If you want to use a secureblue Silverblue image, start with the Fedora Silverblue ISO, Kinoite for Kinoite, and Sericea (Sway Atomic) for Sericea, and all the Wayblue images.
 
-To rebase a [Fedora Atomic](https://fedoraproject.org/atomic-desktops/) installation, follow these steps:
+To rebase a [Fedora Atomic](https://fedoraproject.org/atomic-desktops/) installation, follow these steps<sup>\*</sup>:
 
-*(Important note: the **only** supported tag is `latest`)*
+> [!IMPORTANT]
```
Comment (RoyalOughtness) [inline README.md:86]: this looks way better thank you

File: README.md [hunk 3]
```diff
@@ -78,11 +78,13 @@ Sponsorship options are on the [Donate](DONATE.md) page. All donations are appre
 Have a look at [PREINSTALL-README](PREINSTALL-README.md) before proceeding.
 
 ## Rebasing (Recommended)
-*Note: if you don't already have a Fedora Atomic installation, use a Fedora Atomic iso that matches your secureblue target image to install one. If you want to use a secureblue Silverblue image, start with the Fedora Silverblue iso, Kinoite for Kinoite, and Sericea (Sway Atomic) for Sericea and all the Wayblue images.*
+> [!NOTE] 
+> If you don't already have a Fedora Atomic installation, use a Fedora Atomic ISO that matches your secureblue target image to install one. If you want to use a secureblue Silverblue image, start with the Fedora Silverblue ISO, Kinoite for Kinoite, and Sericea (Sway Atomic) for Sericea, and all the Wayblue images.
 
-To rebase a [Fedora Atomic](https://fedoraproject.org/atomic-desktops/) installation, follow these steps:
+To rebase a [Fedora Atomic](https://fedoraproject.org/atomic-desktops/) installation, follow these steps<sup>\*</sup>:
```
Comment (RoyalOughtness) [inline README.md:84]: why the asterisk?

Comment (mintpilo) [reply README.md:84]: Continuity for the footnote at the end of the section.

File: POSTINSTALL-README.md (hunk 1 above)
Comment (mintpilo) [reply POSTINSTALL-README.md]: I agree that's better wording, committed the change

**Pull request merged**